tests/taskmodules/test_re_text_classification_with_indices.py

```diff
@@ -1043,10 +1043,17 @@ def test_collate_with_add_argument_indices(batch_with_argument_indices):
     torch.testing.assert_close(inputs["pooler_end_indices"], torch.tensor([[6, 11], [9, 14]]))
 
 
-def test_encode_input_multiple_relations_for_same_arguments(caplog):
+@pytest.mark.parametrize(
+    "handle_relations_with_same_arguments", ["keep_first", "keep_none", "keep_second"]
+)
+def test_encode_input_multiple_relations_for_same_arguments(
```

Comment:
please add another test 
```
@pytest.mark.parametrize(
    "handle_relations_with_same_arguments", ["keep_first", "keep_none"]
)
def test_encode_input_duplicated_relations(caplog, handle_relations_with_same_arguments)
```
that tries a document with just one, but duplicated relation.